.github/workflows/test-benchmark-collection-models.yml

```diff
@@ -0,0 +1,54 @@
+name: Benchmark collection
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - dw_misc
```

Comment:
to be deleted then

Reply:
Will leave it until merged, to confirm things are still working. Shouldn't hurt too much.